applications/ConvectionDiffusionApplication/python_scripts/pfem2_convection_process.py

```diff
@@ -0,0 +1,86 @@
+import KratosMultiphysics as KM
+import KratosMultiphysics.ConvectionDiffusionApplication as CDA
+
+def Factory(settings, Model):
+    if not isinstance(settings, KM.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return PFEM2ConvectionProcess(Model, settings["Parameters"])
+
+
+class PFEM2ConvectionProcess(KM.Process):
```

Comment:
why do you add this in the ConDiffApp if it is intended for PFEM2?

Reply:
This process is an interface for an existing utility en ConvectionDiffusionApplication. It doesn't require the compilation of pfem2 application.

Reply:
but why do you call it `PFEM...`?
just curious

Reply:
I agree with @philbucher . This is just an interface to a utility in the `ConvectionDiffusionApplication`. To call it `PFEMWhatEver` is misleading. You can either save it in your application or call it `ApplyMoveParticleUtilityProcess`.